slurm/2_ApplicationSpecific/VASP-GPU-podman/build_vasp.bash

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+if [ "${1}" = "" ] || [ ! -d "${1}" ] || [ "${2}" = "" ] || [ ! -d "${2}" ]
+then
+  echo "usage: ${0} image_tarball_directory bin_directory" >&2
+  echo "e.g." >&2
+  echo "${0} /projects/academic/mygroup/podman_images/ /projects/academic/mygroup/VASP/bin" >&2
+  exit 1
+fi
+
+tarball_dir="${1}"
```

Comment:
The naming convention here confused me, as VASP comes tarred up as well. Maybe we should refer to this variable as something like oci_archive_dir, something to clarify that this directory is meant to house the saved OCI filesystem from the container.

Reply:
done